notebook/static/notebook/js/actions.js

```diff
@@ -131,7 +137,7 @@ define(function(require){
                 env.notebook.execute_cells_below();
             }
         },
-        'go-to-command-mode': {
+        'enter-command-mode': {
```

Comment:
Modes are like contexts, like rooms in a house. You enter rooms, you enter modes.